docs/source/specification.md

```diff
@@ -63,18 +63,102 @@ Together, these provide a framework that {ref}`TRE operator <infrastructure_role
+- TREs open to consultation by impacted parties should be auditable, and impacted parties should be part of any decision making processes. This may include the provision of documentation and educational resources for a diverse audience.
+- Access to public sector data should be reviewed by an independent panel and follow agreed-upon governance to ensure projects using this data are in the public benefit, and provide clarity around any commercial access.
+
+#### Observability
+
+##### Statement
+
+Human initiated and automated processes resulting in change within the TRE should be observable.
+
+##### Rationale
+
+System/process observability is key to understanding whether your policies and controls are actually doing what is intended.
+
+It also allows operators to continuously improve their systems and processes, measure their effectiveness, and identify the causes of incidents. Data can also be made available to other parties such as auditors, data subjects and data providers as part of the assurance process.
+
+This applies to both technical systems and policies/processes.
+
+##### Implications
+
+- In order to understand what is happening within the TRE, both automated and human initiated processes should generate sufficient data. Any generated data should follow standards for provenance and transparency for audit trails.
+- Different levels of observability may be needed for different users. Any data collected from an observability perspective should consider the needs of those who will use it, and minimise collection accordingly.
+- There may be ethical and confidential issues to consider when implemeting the observability principle.
+
+#### Standardisation
+
+##### Statement
+
+TREs should adhere to standards or well-known patterns wherever possible.
+
+##### Rationale
+
+Standardisation makes it easier to design, operate, use and understand TREs, and reduces duplication of work.
+This includes making TREs easier to use, deploy, and audit.
+
+TREs should be built in such a way that they do not restrict or prevent interoperability where this may be desirable in future, by identifying and avoiding or removing barriers to interoperability.
+
+Standardisation is also linked to the public trust principle, as a standard approach to TRE provision will make it eaiser for impacted parties to understand how their data will be used within TREs.
+
+##### Implications
+
+- Standards for TREs don't currently exist, and may be developed in the future. TRE teams should be prepared to adapt their provision to fit the latest agreed upon standard.
+- Standards that TREs adhere to may range in scope, including technical, operational and governance requirements. All agreed upon standards should align to ensure there is no contradiction in requirements.
+- Any developments in standards should reuse or incorporate pre-existing content and resources wherever possible.
```

Comment:
```suggestion

```

I don't think it's clear what "developments in standards" means here

Reply:
Happy to remove